Cargo.toml

```diff
@@ -83,6 +85,10 @@ name = "ironcore_alloy"
 name = "uniffi-bindgen"
 path = "uniffi-bindgen.rs"
 
+[[bin]]
+name = "uniffi-bindgen-java"
+path = "uniffi-bindgen-java.rs"
```

Comment:
How come this is generated as a separate binary?

Reply:
Because it is a separate binary.